wave-fe/package.json

```diff
@@ -20,17 +20,19 @@
     "@angular/platform-browser-dynamic": "~9.1.9",
     "@angular/router": "~9.1.9",
     "firebase": "^7.15.0",
+    "focus-visible": "^5.1.0",
     "rxjs": "~6.5.4",
     "tslib": "^1.10.0",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
+    "@angular-builders/custom-webpack": "^9.2.0",
     "@angular-devkit/build-angular": "~0.901.7",
```

Comment:
Perhaps we could remove this `"@angular-devkit/build-angular"` module, pending to Test/Review it.

Reply:
yes, this is an important test

Reply:
Could you please test it if you have some spare time ? 🤔